Gemfile.lock

```diff
@@ -206,7 +198,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  arm64-darwin-21
```

Comment:
I think you can `git checkout main Gemfile.lock` on this -- I don't believe we need to commit this.

Reply:
Although I'm not totally sure what best practice is for folks who are bundling on different machines 🤔